docs/quickstart/assessment/src/main/java/com/learnosity/quickstart/AuthorApp.java

```diff
@@ -0,0 +1,59 @@
+package com.learnosity.quickstart;
+
+import learnositysdk.request.Init;
+import java.util.HashMap;
+import java.util.Map;
+import org.json.JSONObject;
+
+public class AuthorApp extends App
+{
+    public String initOptions(String domain) {
+        Map<String, String> security = createSecurityObject(domain);
+        JSONObject request = createRequestObject();
+        String secret = config.getProperty("consumerSecret");
+
+        try {
+            Init init = new Init("author", security, secret, request);
+            return init.generate();
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "";
+        }
+    }
+
+    Map<String, String> createSecurityObject(String domain) {
+        var secret = new HashMap();
```

Comment:
should this be security instead of secret?

Reply:
done